.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: python
 python:
-  - "3.5"
+  - "3.8"
```

Comment:
I think we should run against 3.6 and 3.8 for now (so we keep compatibility with bionic and focal versions of python).

We can drop 3.6 if we want to use 3.8-specific features, but for now we don't

Reply:
Makes sense. I only made this change because black needs a more recent version than 3.5 but 3.6 will do.